firmware/Core/Src/unit_tests/unit_test_helpers.c

```diff
@@ -1,2 +1,57 @@
 #include "unit_tests/unit_test_helpers.h"
+#include "telecommands/telecommand_args_helpers.h"
+#include "string.h"
 
+uint8_t TEST_EXEC__TCMD_hex_string_to_byte_array(){
+    char normal[9] = "FFFFFFFF\0";
```

Comment:
Please move each of the test cases to right before each unit test, so that you can see everything you need to at once to fully understand the unit test.